tubesize/src/styles/analytics.css

```
@@ -0,0 +1,78 @@
+    padding: 16px 10px 16px 22px;
+    gap: 10px;
+}
+
+.stats-card:hover {
+    background-color: #2a2a2a;
+    cursor: pointer;
+}
+
+.stat-value {
+    color: #f0ece4;
+    font-family: "JetBrains Mono", "SF Mono", "Fira Code", "monospace";
```

Stylelint (17.10.0)

[error] Expected no quotes around "monospace" (font-family-name-quotes)

(font-family-name-quotes)